BROWSER-SUPPORT.md

```diff
@@ -1,7 +1,5 @@
 # Browser support
 
-## Introduction
-
 The UI-Kit team needs to support all of our users, regardless of their device, web browser or other user agent.
 
 Alongside accessibility, extensive browser and device support is a legal requirement under the <a href="https://www.legislation.gov.au/Latest/C2016C00763" rel="external">Disability Discrimination Act (1992)</a>, requiring equal access to information on laws and government programs.
```

Comment:
I'm uncertain about the value of this para @klepas I'm all for promoting our obligation to accessibility (I made sure we start off by saying this, albeit in less direct terms, at line 5).

If we want to add value (using a reference to our legal obligation) it should be provides alongside direction for kit users to implement the kit in an accessible fashion, which is probably best situated elsewhere in the guide.

Also I reckon there's value in being more direct about this in our overall principles, but they should also live elsewhere.